.github/workflows/ci_cd.yml

```diff
@@ -126,11 +141,26 @@ jobs:
 
       - name: Start API, ingest and index test data
         run: just init
+
       - name: Run API tests
         run: just api-test -T
 
+      - name: Print API test logs
+        if: ${{ always() }}
+        run: |
+          just logs > api_logs
```

Comment:
`just logs` without a service name will print the logs of all services, not just `web`. Also this will intermix all logs for all services. IMO it would be clearer to call this separately with individual service names and export them to different files.

Reply:
I agree with Madison on this one. Additionally, we'd have to remember to continue to add the services to the list of logs we pull if more relevant services are added.